src/lib/controllers/ui-controller.js

```diff
@@ -1934,11 +1974,12 @@ export default class UIController {
           customEv = this.options.textQueryTriggerDesktop
       }
     }
+    const lexisModule = this.getModule('lexis')
     if (ev) {
-      this.evc.registerListener(listenerName, selector, this.getSelectedText.bind(this), ev)
+      this.evc.registerListener(listenerName, selector, this.api.lexis.getSelectedText.bind(lexisModule), ev)
```

Comment:
so, do I understand correctly that right now, the getSelectedText handler in the lexis module is just a reference back to the getSelectedText method in UIController? And the intent is for that code eventually to move into the lexis module itself?

Reply:
Yes, that's exactly it. All lexical requests are supposed to go through the Lexis data module and corresponding code from a UI controller will be moved there. So this is a temporary workaround until then.